src/Arcus.Messaging.Abstractions.ServiceBus/MessageHandling/AzureServiceBusMessageRouter.cs

```diff
@@ -208,10 +209,13 @@ protected AzureServiceBusMessageRouter(IServiceProvider serviceProvider, AzureSe
             var isSuccessful = false;
             using (DurationMeasurement measurement = DurationMeasurement.Start())
             using (IServiceScope serviceScope = ServiceProvider.CreateScope())
-            using (UsingMessageCorrelationEnricher(serviceScope.ServiceProvider, correlationInfo))
```

Comment:
What is this `UsingMessageCorrelationEnricher` method and should we remove it, as it seems that we're no longer using it ?

Reply:
It was previously a returned `IDisposable` of Serilog's `LogContext`. It was `protected`, so made it deprecated.